### ISSUE TITLE
Apache2 doesn't start, mod_proxy-html errors.

### ISSUE THREAD

**Issue by virtadpt** [Byzantium/ByzPi#20]
Error message:

```
Syntax error on line 31 of /etc/apache2/conf.d/proxy-html.conf:
Invalid command 'ProxyHTMLLinks', perhaps misspelled or defined by a module
not included in the server configuration.
```

**Comment by virtadpt** [comment]
Need to install the package libapache2-mod-proxy-html from Debian's package repo.

Need to run `a2enmod proxy_html` to enable mod_proxy_html.

```
Syntax error on line 92 of /etc/apache2/conf.d/proxy-html.conf:
Invalid command 'ProxyRequests', perhaps misspelled or defined by a module
not included in the server configuration.
```

**Comment by virtadpt** [comment]
Added libapache2-mod-proxy-html to the list of packages that need to be installed. Commit: 0c9a69e7ffb5e88baccd5c58c4b93e1710f57e4a

**Comment by virtadpt** [comment]
Recommended fix: `a2enmod proxy` to enable mod_proxy.

Source: http://last-error-message.blogspot.com/2011/02/invalid-command-proxyrequests.html

**Comment by virtadpt** [comment]
`sudo a2enmod proxy` was successful.  New error message:

```
Syntax error on line 111 of /etc/apache2/conf.d/proxy-html.conf:
Invalid command 'RequestHeader', perhaps misspelled or defined by a module not included in the
server configuration
Action 'start' failed.
```

**Comment by virtadpt** [comment]
`sudo a2enmod headers` was successful.  New error message:

```
(98)Address already in use: make_sock: could not bind to address 0.0.0.0:443
no listening sockets available, shutting down
```

Fixed in commit 89b545ddc573b139a5b38f6309b7cb2a10823150.

**Comment by virtadpt** [comment]
New error message:

```
The Apache error log may have more information.
failed!
```

/var/log/apache2/error.log says:

```
(2)No such file or directory: apache2: could not open error log file /var/log/httpd/ssl_error_log.
Unable to open logs
```

That needs to be /var/log/apache2.

Fixed in commit 5a9c88cc2a6c5b53537a0f28b410f1ab9ba98a35.

**Comment by virtadpt** [comment]
New problem.  In /var/log/apache2/ssl_error_log:

```
[Mon Sep 16 19:18:19 2013] [error] (2)No such file or directory: could not open transfer log file /var/log/httpd/ssl_request_log.
```

Fixed in commit b376ff6e6eca85cb7dd306ef86c4fc99b3f1adf4.

**Comment by virtadpt** [comment]
One more bug, though it doesn't prevent Apache from starting:

```
Starting web server: apache2Warning: DocumentRoot [/srv/httpd/htdocs] does not exist
```

/srv/httpd/htdocs does not, in fact, exist.

**Comment by virtadpt** [comment]
Fixed by editing /etc/apache2/conf.d/httpd-ssl.conf, directive DocumentRoot.  Should be set to "/var/www".

Fixed in commit 88637a946918931aa2e2cdb1b95427a446d26417.

Works now.
